RockPaperScissors/RockPaperScissors/MukZziPpa.swift

```diff
@@ -0,0 +1,92 @@
+//
+//  RockPaperScissors - main.swift
+//  Created by Lingo, 우롱차.
+//  Copyright © yagom academy. All rights reserved.
+//
+
+import Foundation
+
+enum MukZziPpaHandType: Int, CaseIterable {
+    case muk = 1
+    case zzi = 2
+    case ppa = 3
+    
+    static var randomPick: Int? {
+        return Self.allCases.randomElement()?.rawValue
+    }
+}
+
+func getPlayerMukZziPpaInput() -> Int? {
+    guard let playerInput = readLine(),
+          let playerIntInput = Int(playerInput)
+    else {
+        return nil
+    }
+    
+    if playerIntInput == Settings.exitCode {
+        return playerIntInput
+    }
+    
+    guard let _ = MukZziPpaHandType(rawValue: playerIntInput) else {
+        return nil
+    }
+
+    return playerIntInput
+}
+
+func printMukZziPpaGameMenu(_ result: GameResult) {
+    switch result {
+    case .playerWin:
+        print(GameDisplayMessage.playerTurnMukzzippaMenu, terminator: "")
+    case .playerLose:
+        print(GameDisplayMessage.computerTurnMukzzippaMenu, terminator: "")
+    default:
+        break
+    }
+}
+
+func startMukZziPpaGame(_ result: GameResult) -> GameResult? {
+    printMukZziPpaGameMenu(result)
```

Comment:
묵찌빠 게임이 시작할 때는, 이미 이전 게임인 가위바위보에서 승패가 정해진 상황이잖아요?
그래서 묵찌빠 시작하는 메서드에서 `파라미터로 이전 게임의 결과`를 받아주시는 거로 이해했습니다! 👍🏻 

이때, 파라미터 생략하기 보다는, `with` 전치사를 사용해서
"묵찌빠 게임을 어떤 결과와 `함께` 시작한다." -> 이런 늬앙스를 담으면 어떨까요? 😄 

```suggestion
func startMukZziPpaGame(with result: GameResult) -> GameResult? {
    printMukZziPpaGameMenu(result)
```

```swift
// 사용되는 모습
startMukZziPpaGame(with: .playerLose)
startMukZziPpaGame(with: .playerWin)
```